web/packages/shared/components/FieldMultiInput/FieldMultiInput.tsx

```diff
@@ -90,15 +116,19 @@ export function FieldMultiInput({
           // procedure whose complexity would probably outweigh the benefits).
           <Flex key={i} alignItems="center" gap={2}>
             <Box flex="1">
-              <Input
+              <FieldInput
                 value={val}
+                rule={precomputed(
+                  validationResult.items?.[i] ?? { valid: true }
+                )}
```

Comment:
Could you help me understand how this is interacting with default rule? The default rule just returns `valid: true`, why would we need to do extra validation here based on item length? could we just return whatever the default rule returns here instead of `valid: true`?

Reply:
This `{ valid: true }` object is a per-item object, as opposed to the one returned from the default rule, which applies to *all* items.